src/apprt/gtk/App.zig

```diff
@@ -268,6 +269,8 @@ pub fn init(core_app: *CoreApp, opts: Options) !App {
         std.posix.exit(1);
     };
 
+    try i18n.init(core_app.alloc);
```

Comment:
A comment here that explains why this has to happen here in ordering and what exactly this is doing would be helpful.

Reply:
I kinda just put it in an arbitrary place - *ideally* you want to initialize it before any localized messages can appear, and since we don't have any localized log messages you can put it anywhere really

Reply:
Yeah I don't particularly care, I think its right where it is, but a comment explaining what this does and why here is helpful to future maintenance.